### PR TITLE
Yet more rubocop fixes

### PR DESCRIPTION
**Pull request by mpw5**
#### What

Fixes offences for a number of rubocop cops:

`RSpec/RepeatedDescription`
`RSpec/RepeatedExample`
`RSpec/RepeatedExampleGroupBody`
`RSpec/RepeatedExampleGroupDescription`
`RSpec/IteratedExpectation`
`Rails/Blank`

Also fixes line length issues in the files affected by the above.

#### Why

To improve overall code quality

### CODE REVIEW

File: spec/models/claim/base_claim_spec.rb [hunk 1]
```diff
@@ -220,14 +223,16 @@ class MockSteppableClaim < Claim::BaseClaim
...
 
-    describe '#disbursements' do
-      it 'return the sum of the amounts for the disbursements without vat' do
-        expect(@claim.disbursements_without_vat_net).to eq 125.0
+      context 'when the disbursment does not include vat' do
+        it 'return the sum of the amounts for the disbursements without vat' do
+          expect(@claim.disbursements_without_vat_net).to eq 125.0
+        end
       end
     end
```
Comment (jrmhaig) [inline spec/models/claim/base_claim_spec.rb]: For consistency with `describe` names have you considered;

```ruby
    describe '#disbursements_with_vat_net' do
      subject { @claim.disbursements_with_vat_net }

      it { is_expected.to eq 150.50 }
    end

    describe '#disbursements_without_vat_net' do
      subject { @claim.disbursements_without_vat_net }

      it { is_expected.to eq 125.0 }
    end
```

File: spec/validators/expense_validator_spec.rb [hunk 2]
```diff
@@ -444,51 +460,74 @@
     end
   end
 
-  describe 'validate_mileage_rate_id' do
+  describe 'distance' do
```
Comment (jrmhaig) [inline spec/validators/expense_validator_spec.rb]: Should this be:

```suggestion
  describe '#distance' do
```

Comment (mpw5) [reply spec/validators/expense_validator_spec.rb]: 👍

File: spec/models/claim/base_claim_spec.rb (hunk 1 above)
Comment (mpw5) [reply spec/models/claim/base_claim_spec.rb]: Good call. I've also extended that format to the two tests above so the whole `disbursements` context is consistent.

**Review by jrmhaig — APPROVED** [review]

**Pull request merged**